events-api/src/modules/tickets/tickets.service.ts

```diff
@@ -0,0 +1,20 @@
+import { NotFoundError } from "#/shared/errors/notFoundError";
+import { TicketsCreateDto } from "./dto/requests/tickets-create-dto";
+import { TicketModel } from "./tickets.model";
+import { TicketsRepository } from "./tickets.repository";
+
+export class TicketsService {
+  static async createTicket(newTicket: TicketsCreateDto): Promise<TicketModel> {
+    return await TicketsRepository.create(newTicket);
+  }
+
+  static async getTicket(id: string): Promise<TicketModel | TicketModel[]> {
+    const res = await TicketsRepository.getTicketById(id);
+
+    if (Array.isArray(res) && res.length === 0) {
```

Comment:
If you will implement the logic of retrieving a single object from the array from res in the repository, you don't need to have these checks on the service level